modules/music/api/music-routes.ts

```diff
@@ -25,4 +34,22 @@ module.exports = function(app : core.Express){
            total: data.length
         })
     })
+
+    app.get("/album/", (req: Request, res: Response) => {
+        let data: Album[] = []
```

Comment:
Are you able to add a qty in the url to help define the amount of songs that are returned in the playlist. :)